Makefile

```diff
@@ -380,16 +397,12 @@ release-publish: release-prereqs $(UPLOAD_DIR) helm-index
 	@echo ""
 	@echo "  https://github.com/projectcalico/calico/releases/tag/$(CALICO_VER)"
 	@echo ""
+	$(MAKE) helm-index
```

Comment:
nit: I think this should probably come before the call to `helm-index` so that the message for verifying the tags will be one of the last things the user sees when they run the target.

Reply:
Agreed.